### PR TITLE
Fixed build errors regarding darknet libraries.

### PR DESCRIPTION
**Pull request by sanskar-sopho**
FIND_PATH was unable to find the darknet libraries, so I added their most certain path in PATHS.

### CODE REVIEW

**Comment by vinay0410** [comment]
Did you, specify, darknet installation directory in -DDARKNET_PATH ?

**Comment by sanskar-sopho** [comment]
Obviously, still it was not able to find.

**Comment by vinay0410** [comment]
Could you please list the contents of you darknet installation directory, passed in ```-DDARKNET_PATH```, because you are actually removing include subdirectory in cmake.
What if someone's headers are in include, for instance, I passed the installation dir ```/usr/local``` and my DarkNetAPI folder is in ```/usr/local/include```

**Comment by sanskar-sopho** [comment]
@vinay0410 , I am removing ```include``` as ```FIND_PATH``` is searching for NAMES ```DarknetAPI/DarknetAPI.h``` and not ```DarknetAPI.h```. I think ```FIND_PATH``` should search within all the subfolders but somehow it does not. Talking about generalization, people are supposed to fork the darknet repository which has ```darknetAPILib darknetExamplesLib darknetLib``` libraries in the folders I have included. I think this will work fine. Please correct me if I am wrong.

**Comment by vinay0410** [comment]
You are **not** including the installation path of darknet, rather the path to actual repository which contains folders such as ```src```, ```examples``` and ```Darknet API```. That's the only plausible reason why your solution is working. Your installation path while installing darknet should be something like this ```/usr/local/```. This will put header files in ```/usr/local/include/``` and libraries in ```/usr/local/lib/```. You will have something like this
image
You can see for yourself all the libraries you mentioned are available under ```use/local/lib/```

File: DeepLearningSuite/CMakeLists.txt [hunk 1]
```diff
@@ -93,12 +93,12 @@ IF (NOT DARKNET_PATH)
...
 ENDIF()
 
-FIND_PATH( DARKNET_INCLUDE_DIR NAMES DarknetAPI/DarknetAPI.h  PATHS ${DARKNET_PATH}/include )
+FIND_PATH( DARKNET_INCLUDE_DIR NAMES DarknetAPI/DarknetAPI.h  PATHS ${DARKNET_PATH} )
 SET(DARKNET_NAMES darknetAPILib darknetExamplesLib darknetLib)
 #SET(DARKNET_NAMES darknetLib)
 
  FOREACH(N ${DARKNET_NAMES})
-	FIND_LIBRARY( DARKNET_LIBRARIES_${N} NAMES ${N} PATHS ${DARKNET_PATH}/lib )
+	FIND_LIBRARY( DARKNET_LIBRARIES_${N} NAMES ${N} PATHS ${DARKNET_INCLUDE_DIR}/DarknetAPI/ ${DARKNET_INCLUDE_DIR}/src/ ${DARKNET_INCLUDE_DIR}/examples)
```
Comment (vinay0410) [inline DeepLearningSuite/CMakeLists.txt]: This directory structure is in the source repository of jderobot fork, you would only need this if your are linking libs from the source repository build.
And that's not how it is supposed to be, libraries are supposed to be linked from the installation directory not the source repository

Comment (sanskar-sopho) [reply DeepLearningSuite/CMakeLists.txt]: @vinay0410 Thanks a lot, I got your point.

Comment (vinay0410) [reply DeepLearningSuite/CMakeLists.txt]: Great !!!

**Comment by chanfr** [comment]
Sorry about my delay...

sure @vinay0410 was right it assumes that you are setting the installation path not the source's.

**Pull request closed without merging**